apps/earn-protocol-institutions/features/admin/AdminPanelFeedbackList.module.css

```
@@ -115,91 +299,163 @@
 
 .replyTextarea {
   width: 100%;
   min-height: 120px;
-  padding: 10px;
-  border-radius: 10px;
-  border: 1px solid #1f2937;
+  padding: 12px;
+  border-radius: 8px;
+  border: 1px solid #374151;
   color: #e5e7eb;
+  font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)